app/controllers/api/v0/articles_controller.rb

```diff
@@ -21,11 +18,13 @@ class ArticlesController < ApiController
       def index
         @articles = ArticleApiIndexService.new(params).get
 
-        set_surrogate_key_header "articles", @articles.map(&:record_key)
+        set_surrogate_key_header Article.table_key, @articles.map(&:record_key)
       end
 
       def show
         @article = Article.published.includes(:user).find(params[:id]).decorate
+
+        set_surrogate_key_header @article.record_key
```

Comment:
this will be purged by 

https://github.com/thepracticaldev/dev.to/blob/7d93594447afc8a341a777099c39c854404b5a3c/app/models/article.rb#L649-L651

which will eventually call `article.purge` on this article when the article is modified